h2o-r/h2o-package/R/gam.R

```diff
@@ -15,7 +15,8 @@
 #'        The response must be either a numeric or a categorical/factor variable. 
 #'        If the response is numeric, then a regression model will be trained, otherwise it will train a classification model.
 #' @param training_frame Id of the training data frame.
-#' @param gam_columns Predictor column names for gam
+#' @param gam_columns Arrays of predictor column names for gam for smoothers using single or multiple predictors like
+#'        {{'c1'},{'c2','c3'},{'c4'},...}
```

Comment:
@wendycwong I know that this text generated from the Java docs using `gen_gam.py`, but I wonder if we can post-process to if from: `{{'c1'},{'c2','c3'},{'c4'},...}` to use R code?  As a user, I might get confused and perhaps not know if it should be specified as:  `c(c("c1"),c("c2", " c3"), c("c4")))` or `list(c("c1"),c("c2", " c3"), c("c4")))`.  Since it's list in R and Python, we could say "list of predictor column names" vs "arrays of predictor column names" (I think we have a precedent of describing things in the API as "lists" even if they are technically arrays in Java, for clarity to the R/Py users).